### DISCUSSION TITLE
Redirect Old Public Recipe Browser Link

### DISCUSSION THREAD

**Discussion by phellarv** [mealie-recipes/mealie#2745] [Feature Request]
### First Check

- [X] This is not a feature request.
- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the Mealie documentation, with the integrated search.
- [X] I already read the docs and didn't find an answer.
- [ ] This issue can be replicated on the demo site (https://demo.mealie.io/).

### What is the issue you are experiencing?

The feature from michael-genson:feat/public-recipe-browser #2525 is not working anymore. Not sure when it did go away, but sometime after 11.10.23 and today

### Steps to Reproduce

Go to http(s)://mealie.url/user/profile and observe that the "Get public link"-button is missing.
![2023-11-20_22-52_1](https://github.com/mealie-recipes/mealie/assets/822762/f104b5a8-f8ff-447a-aab2-b35509900ad0)
![2023-11-20_22-52](https://github.com/mealie-recipes/mealie/assets/822762/f81d6305-6a24-445c-bda3-3a39867bb924)

Group is not set to Private

### Please provide relevant logs

I have no logs for the incident.

### Mealie Version

image: ghcr.io/mealie-recipes/mealie:nightly
Version: nightly
Build: 08656ec09ac58e5156bdd585079b67d1c97013ba
Application Mode: Production
Demo Status: Not Demo
API Port: 9000
API Docs: Enabled
Database Type: postgres
Database URL: postgresql://*****:*****@postgres:5432/mealie
Default Group: Home
Recipe Scraper Version: 14.51.0 

### Deployment

Docker (Linux)

**Comment by michael-genson** [comment]
Instead of having two sets of pages (private pages and public "explore" pages) all pages are just public now (provided your group isn't private). More info here: https://github.com/mealie-recipes/mealie/pull/2632

Or, put another way, copy your recipes page URL and try going there logged-out (or in incognito) and it should work!

**Comment by phellarv** [comment]
Aaah - OK. Then would it be possible that `http://mealie.url/explore/recipes` can automatically redirect to `http://mealie.url/g/<default-group>` ?

**Comment by michael-genson** [comment]
I think you mean the old URL `/explore/your-group/recipes`, `/explore/recipes` was never a valid URL. I think that's something we could add, though
